Library/Homebrew/download_strategy.rb

```diff
@@ -316,6 +329,13 @@ def ext
   end
 end
 
+# It is a superclass for the following strategies:
+# CurlApacheMirrorDownloadStrategy, CurlPostDownloadStrategy,
+# NoUnzipCurlDownloadStrategy, CurlBottleDownloadStrategy, S3DownloadStrategy
+# and GitHubPrivateRepositoryDownloadStrategy.
+# Files downloaded by this way follow the naming convention
+# name-version.extension . Also the incomplete file is
+# name-version.extension.incomplete .
 class CurlDownloadStrategy < AbstractFileDownloadStrategy
```

Comment:
Can probably skip this comment. Don't want to name specific classes or behaviour as the documentation will become outdated if the code changes.